velox/vector/arrow/Bridge.cpp

```diff
@@ -1112,7 +1114,7 @@ void exportConstantValue(
         ? sizeof(StringView)
         : vec.type()->cppSizeInBytes();
 
-    valuesVector = VELOX_DYNAMIC_SCALAR_TYPE_DISPATCH(
+    valuesVector = VELOX_DYNAMIC_SCALAR_TYPE_DISPATCH_ALL(
```

Comment:
This change extends the support for UNKNOWN and OPAQUE types. Should `OPAQUE` type be excluded?